src/test/java/com/snowflake/kafka/connector/internal/streaming/SnowflakeSinkServiceV2IT.java

```diff
@@ -579,6 +581,89 @@ public void testNativeAvroInputIngestion() throws Exception {
     service.closeAll();
   }
 
+  @Test
+  public void testTableCreationAndNativeAvroInputIngestionWithSchematization() throws Exception {
+    Map<String, String> config = TestUtils.getConfForStreaming();
+    config.put(SnowflakeSinkConnectorConfig.ENABLE_SCHEMATIZATION_CONFIG, "true");
+    config.put(
+        SnowflakeSinkConnectorConfig.VALUE_CONVERTER_CONFIG_FIELD,
+        "io.confluent.connect.avro.AvroConverter");
+    config.put("value.converter.schema.registry.url", "http://fake-url");
+    SnowflakeSinkConnectorConfig.setDefaultValues(config);
+    // avro
+    SchemaBuilder schemaBuilder =
+        SchemaBuilder.struct()
+            .field("id", Schema.INT32_SCHEMA)
+            .field("first_name", Schema.STRING_SCHEMA)
+            .field("rating", Schema.FLOAT32_SCHEMA)
+            .field("approval", Schema.BOOLEAN_SCHEMA)
+            .field(
+                "info_map", SchemaBuilder.map(Schema.STRING_SCHEMA, Schema.INT32_SCHEMA).build());
+    Struct original =
+        new Struct(schemaBuilder.build())
+            .put("id", 42)
+            .put("first_name", "zekai")
+            .put("rating", 0.99f)
+            .put("approval", true)
+            .put("info_map", Collections.singletonMap("field", 3));
+
+    String schemaString =
+        "{\n"
+            + "            \"type\":\"record\",\n"
+            + "            \"name\":\"value_schema\",\n"
+            + "            \"fields\":[\n"
+            + "                {\"name\":\"id\",\"type\":\"int\"},\n"
+            + "                {\"name\":\"first_name\",\"type\":\"string\"},\n"
+            + "                {\"name\":\"rating\",\"type\":\"float\"},\n"
+            + "                {\"name\":\"approval\",\"type\":\"boolean\"},\n"
+            + "               "
+            + " {\"name\":\"info_map\",\"type\":{\"type\":\"map\",\"values\":\"string\"}}\n"
+            + "            ]\n"
+            + "        }";
```

Comment:
If you could extra this to some field at class level or in constants. 

Also kudos for this schema, I know how boring and convoluted this struct and schema field is.